src/lint/lintFolder.spec.ts

```diff
@@ -71,11 +77,47 @@ const expectedDiagnostics = [
 
 describe('lintFolder', () => {
   it('should identify lint issues in a given folder', async () => {
-    const results = await lintFolder(path.join(__dirname, '..'))
-
+    await createFolder(path.join(__dirname, 'lint-folder-test'))
+    const content = await readFile(
+      path.join(__dirname, '..', 'Example File.sas')
+    )
+    await createFile(
+      path.join(__dirname, 'lint-folder-test', 'Example File.sas'),
+      content
+    )
```

Comment:
We can use also `copy` instead of reading/writing.

NOTE: need to move copy from `sasjs/cli/utils/file` to `sasjs/utils/file` first